Podfile

```
@@ -43,7 +43,7 @@ target 'WooCommerce' do
   pod 'WordPressAuthenticator', '~> 2.0.0-beta.1'
   # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :commit => ''
   # pod 'WordPressAuthenticator', :git => 'https://github.com/wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => ''
-  # pod 'WordPressAuthenticator', :path => '../WordPressAuthenticator-iOS'
+  # pod 'WordPressAuthenticator', :path => '~/WordPressAuthenticator-iOS'
```

nit: it'd be nice not having diffs on the `Podfile` when there are no pod changes, since any diffs change the checksum in `Podfile.lock` below 🙂 If you revert the change on this line then run `bundle exec pod install` again, the diff `Podfile.lock` should be back to the value in `trunk` too

```suggestion
  # pod 'WordPressAuthenticator', :path => '../WordPressAuthenticator-iOS'
```